src/main/java/CommandManager.java

```diff
@@ -0,0 +1,91 @@
+import java.util.Scanner;
+
+
+public class CommandManager {
+    /**
+     * Scanner to get user input
+     */
+    private static Scanner sc = new Scanner(System.in);
+    /**
+     * Manage the reminder function
+     */
+    private static TaskManager TaskManager;
```

Comment:
Why is there a 2 "TaskManager"s?

Reply:
Deleted, in previous commits methods in TaskManager were not static. This variable is actually never used.

Reply:
> Why is there a 2 "TaskManager"s?

Deleted, in previous commits methods in TaskManager were not static. This variable is actually never used. Thanks for reviewing.